src/content/docs/integrations/kubernetes-integration/installation/install-fargate-integration.mdx

```diff
@@ -0,0 +1,392 @@
+                fsGroup: 1001
+                runAsUser: 1001
+              serviceAccount: ksm-kube-state-metrics
+              serviceAccountName: ksm-kube-state-metrics
+              terminationGracePeriodSeconds: 30
+              volumes:
+              - emptyDir: {}
+                name: tmpfs-data
+              - emptyDir: {}
+                name: tmpfs-user-data
+              - emptyDir: {}
+                name: tmpfs-tmp
+              - emptyDir: {}
+                name: tmpfs-cache
+        ```
+  
+  </Collapser>
+</CollapserGroup>
+
+3. Create a `ClusterRoleBinding`, or add to a previously created one the `ServiceAccount` of the application that is going to be monitored. All the workloads may share the same `ClusterRoleBinding`, but the `ServiceAccount` of each one must be added to it.
+4. Create a secret containing the New Relic license key. Each namespace needs its own secret.
+
+#### Configuration [#config-manual]
+
+When adding the manifest of the sidecar agent manually, you can use any agent configuration option to configure the agent behavior.
+**NEED A LINK HERE**
+
+#### Update to the latest version
+
+To update any of the components, you just need to modify the deployed yaml. 
+
+Updating any of the fields of the injected container will cause the pod to be re-created.
+
+<Callout variant="important">
+The agent cannot hot load the New Relic license key. After updating the secret, you need to rollout the deployments again.
+</Callout>
+
+#### Uninstall the Fargate integration
+
+To remove the injected container and the related resources, you just have  to remove the following:
+* The sidecar from the workloads that should be no longer monitored 
+* All the secrets containing the newrelic license 
+* `ClusterRole` and `ClusterRoleBinding` objects.
+
+Notice that removing the sidecar container will cause the pod to be re-created.
+
+## Troubleshooting
+
+Here are some tips if you have problems:
+
+### DaemonSet replicas are being deployed into Fargate nodes
+
+If you notice that any Infra DaemonSet replicas are being scheduled on Fargate nodes, it might be because the nodeAffinity rules are not configured properly.
+
+Double-check that the solution was installed with the global.fargate option to true, either through the command line (--set global.fargate=true) or in the `values.yaml` file. If the installation method was not Helm, you’ll need to manually add nodeAffinity rules to exclude fargate nodes.
+
+### Garbage collection
+
+Currently even if the `ClusterRoleBinding` gets updated automatically, there is no controller that watches the whole cluster making sure that the `ClusterRoleBinding` is in sync with the current status. Therefore, if a subject is no longer needed, no automation garbage collects it. If the object is deleted, no automation is in place to recreate it with the needed subjects. **I MOVED THIS OUT OF THE LIMITATIONS SECTION BECAUSE OF A REVIEW COMMENT. IS THIS APPROPRIATE HERE?**
```

Comment:
I would get rid of it for now. It should not be a common use-case and the user should expect things to breaks if clusterRoleBindings are deleted manually

Reply:
@paologallinaharbur, OK, this will be deleted.